README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
```

Comment:
No, I'm happy with Rotations.jl, I simply meant you could change the docs to be a little more specific, since you only have 3D rotations here at the moment.